ChangeLog

```diff
@@ -1,3 +1,7 @@
+2022-09-23  Kevin Ushey  <kevinushey@gmail.com>
+
+    * R/Module.R: Force 'finalize' method to be materialized
```

Comment:
Old Emacs roots here: 1 tab or eight spaces, please.  

(Not sure editorconfig has a mode for ChangeLog...)

Reply:
Sorry, will fix that up! RStudio wasn't smart enough to infer the indentation here...

Reply:
If only we knew someone working on the IDE :joy_cat: